config/ac_oidc.go

```diff
@@ -24,35 +29,74 @@ type OIDC struct {
 	ConfigurationTTL        string   `hcl:"configuration_ttl,optional"`
 	VerifierMethod          string   `hcl:"verifier_method,optional"`
 
+	// configuration related backends
+	AuthorizationBackendName       string `hcl:"authorization_backend,optional"`
+	ConfigurationBackendName       string `hcl:"configuration_backend,optional"`
+	DeviceAuthorizationBackendName string `hcl:"device_authorization_backend,optional"`
```

Comment:
Unused, can be removed.

Reply:
In fact those fields are read at https://github.com/avenga/couper/pull/433/files#diff-4cdd6267fa8b885e1f5bff7cf63d185ad4bc4b3cc348200b5aeb0e0803c05dc5R50
Applies to all ac with same pattern.

Still missing more specific tests for this kind of reference / overrides.

Reply:
We don't use either token revocation or the device authorization flow.

Reply:
Got it. Removed.